spec/presenters/flood_risk_engine/address_presenter_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+module FloodRiskEngine
+  RSpec.describe AddressPresenter, type: :presenter do
+    let(:subject) { AddressPresenter.new(address) }
+
+    describe "to_single_line" do
+      context "when all address parts are present" do
+        let(:address) {
+          Address.new(premises: "a",
+                      street_address: "b",
+                      locality: "c",
+                      city: "d",
+                      postcode: "e")
+        }
+        it "returns the address as a comma delimited string" do
+          expect(subject.to_single_line).to eq("a,b,c,d,e")
```

Comment:
No space between each one? I'd expect the output to be `"a, b, c, d, e"` or is there a requirement to be more compact.

`Horizon House,Deanery Rd,Bristol BS1`

or

`Horizon House, Deanery Rd, Bristol BS1`

I think the second is easier to read.